### PR TITLE
BAEL-2840 - Spring Data Web Support

### PR DESCRIPTION
**Pull request by alejandrogervasio**

### CODE REVIEW

File: spring-data-rest/src/main/java/com/baeldung/springdatawebsupport/application/repositories/UserRepository.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import com.baeldung.springdatawebsupport.application.entities.User;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.querydsl.QuerydslPredicateExecutor;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends PagingAndSortingRepository<User, Long>,
+        QuerydslPredicateExecutor<User> {
```
Comment (KevinGilmore) [inline spring-data-rest/src/main/java/com/baeldung/springdatawebsupport/application/repositories/UserRepository.java]:
```suggestion
  QuerydslPredicateExecutor<User> {
```

Comment (KevinGilmore) [reply spring-data-rest/src/main/java/com/baeldung/springdatawebsupport/application/repositories/UserRepository.java]: indentation

File: spring-data-rest/src/test/java/com/baeldung/springdatawebsupport/application/test/UserControllerIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    @Test
+    public void whenUserControllerInjected_thenNotNull() throws Exception {
+        assertThat(userController).isNotNull();
+    }
+
+    @Test
+    public void whenGetRequestToUsersEndPointWithIdPathVariable_thenCorrectResponse() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/users/{id}", "1")
+                .contentType(MediaType.APPLICATION_JSON_UTF8))
```
Comment (KevinGilmore) [inline spring-data-rest/src/test/java/com/baeldung/springdatawebsupport/application/test/UserControllerIntegrationTest.java]: Let's adjust the indentation in here as well.

Comment (alejandrogervasio) [reply spring-data-rest/src/test/java/com/baeldung/springdatawebsupport/application/test/UserControllerIntegrationTest.java]: Sure. I'll fix the indentation here too.

File: spring-data-rest/src/main/java/com/baeldung/springdatawebsupport/application/repositories/UserRepository.java (hunk 1 above)
Comment (alejandrogervasio) [reply spring-data-rest/src/main/java/com/baeldung/springdatawebsupport/application/repositories/UserRepository.java]: No problem. I fixed the indentation in here as well.

**Pull request merged**